constructor/nsis/_system_path.py

```diff
@@ -73,7 +73,7 @@ def remove_from_system_path(pathname, allusers=True, path_env_var='PATH'):
         reg_value = None
         try:
             reg_value = reg.QueryValueEx(key, path_env_var)
-        except WindowsError:
```

Comment:
Same thing as `_nsis.py`. Can't touch this.

Reply:
excluded